test/lib/y2users/autoinst/reader_test.rb

```diff
@@ -49,6 +49,14 @@
       expect(root_user.password.account_expiration.expire?).to eq(false)
 
       expect(config.login?).to eq(false)
+
+      user_defaults = config.user_defaults
+      expect(user_defaults.expiration).to eq nil
+      expect(user_defaults.inactivity_period).to eq(-1)
```

Comment:
This kind of values are strings in `Password` class. Could it be a problem?

Reply:
I don't think so, but I will double-check